examples/third_party/apr_util/BUILD.expat.bazel

```diff
@@ -39,6 +39,7 @@ configure_make(
         "--without-docbook",
         "--without-examples",
         "--without-tests",
+        "CFLAGS='-fPIC'",
```

Comment:
I don't love that this is required. Does `features=["pic"]` work for this?

Reply:
I tried `features=["pic"]` and `-fPIC` was not added to the compiler command line args :(